content/en/apps/guides/performance/telemetry.md

```diff
@@ -39,8 +39,13 @@ The telemetry data gathered changes with different versions of the framework. Cu
 | `boot_time` | The overall boot time including loading the code, purging, and accessing the database. |
 | `boot_time:1:to_first_code_execution` | The time between the page loading and the JavaScript starting to run. |
 | `boot_time:2:to_bootstrap` | The time between JavaScript starting and the bootstrapping (purging, initial replication, etc) to complete. |
+| `boot_time:2_1:to_replication` | The time it takes to complete the replication. Added in 3.14. |
+| `boot_time:2_2:to_purge` | The time it takes to complete the DB purge. Added in 3.14. |
+| `boot_time:2_3:to_purge_meta` | The time it takes to complete the Meta purge. Added in 3.14. |
 | `boot_time:3:to_angular_bootstrap` | The time between bootstrapping completing and the webapp being ready to use. |
 | `boot_time:4:to_db_warmed` | The time between the webapp being ready to use and the database being ready to use. |
+| `boot_time:purgedDB:<boolean>` | `boot_time:purgedDB:true` if DB was purged, `boot_time:purgedDB:false` if DB wasn't purged. Added in 3.14. |
+| `boot_time:purgedMeta:<boolean>` | `boot_time:purgedMeta:true` if Meta was purged, `boot_time:purgedMeta:false` if Meta wasn't purged. Added in 3.14. |
```

Comment:
As above, what is `Meta` here for the reader? Also, all the other telemetry entries are snake_cased, while this one is camelCased. 
```suggestion
| `boot_time:purging_meta:<boolean>` | `boot_time:purging_meta:true` when purging of the local meta database ran, `boot_time:purging_meta:false` otherwise. Added in 3.14. |
```